examples/src/demo.cr

```diff
@@ -45,14 +45,30 @@ MY_APP_SERVER.config do |app|
   # (HTTP METHODS)[https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html]
   routes :static do
     # Each route is defined as follow
-    # verb, resources : String, controller : Symbol, action : Symbol,
-    # pipeline : Symbol
-    get "/*", StaticController, :index
+    # verb resource : String, controller : Symbol, action : Symbol
+    get "/index.html", StaticController, :index
```

Comment:
What's the intention? 

Also, we should find a way to define a static method/macro that does not need a controller/action to be defined with it just a path.